### PR TITLE
Update ce_vxlan_arp to fix bugs

### PR DESCRIPTION
**Pull request by yanzhangi**
##### SUMMARY

Modifying regular expressions

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ansible/modules/network/cloudengine/ce_vxlan_arp.py
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by jctanner** [comment]
bot_status

File: lib/ansible/modules/network/cloudengine/ce_vxlan_arp.py [hunk 1]
```diff
@@ -286,12 +286,12 @@ def get_current_config(self):
...
         flags = list()
-        exp = "| ignore-case section include evn bgp|host collect protocol bgp"
+        exp = r"| ignore-case section include evn bgp|host collect protocol bgp"
         if self.vbdif_name:
-            exp += "|^interface %s$" % self.vbdif_name
+            exp += r"|^#\s+interface %s\s+" % self.vbdif_name.lower().capitalize().replace(" ", "")
 
         if self.bridge_domain_id:
-            exp += "|^bridge-domain %s$" % self.bridge_domain_id
+            exp += "|^#\s+bridge-domain %s\s+" % self.bridge_domain_id
```
Comment (yuandongx) [inline lib/ansible/modules/network/cloudengine/ce_vxlan_arp.py]: miss 'r'.

**Comment by hwDCN** [comment]
LGMT

**Review by yuandongx — APPROVED** [review]
+1

**Comment by hwDCN** [comment]
+1

**Pull request merged**